docs/tutorials/scenario_management/6_scenario_comparison/index.md

```diff
@@ -10,129 +10,182 @@ img: images/icon-code.svg
 
 ![Configuration](images/config.svg){ width=90% : .tp-image-border }
 
-Taipy offers a way to compare data nodes across scenarios by including a function directly in the configuration of the scenario.
+Taipy offers a way to compare data nodes across scenarios by including a function directly in the 
+configuration of the scenario. This is particularly useful to visualize the differences between 
+scenarios and make decision upon it.
 
 [Download the code](./src/scenario_comparison.py){: .tp-btn target='blank' }
 
 # Comparing scenarios
 
 **Step 1:** Declare data nodes on which the comparison functions are applied.
 
-In this example, we want to apply a comparison to the '_output_' Data Node. It is indicated in the comparators parameter of the `configure_scenario`.
+In this example, we want to apply a comparison to the '_revenues_' Data Node. It is indicated in 
+the comparators parameter of the `configure_scenario`.
 
 ```python
-scenario_cfg = Config.configure_scenario(id="multiply_scenario",
-                                         name="my_scenario",
-                                         tasks_configs=[first_task_cfg,
-                                                        second_task_cfg],
-                                         comparators={output_data_node_cfg.id: compare_function},)
+# Scenario configuration
+scenario_cfg = Config.configure_scenario(
+    id="pricing_strategy",
+    task_configs=[predict_sales_task_cfg, calculate_revenue_task_cfg],
+    comparators={revenue_output_cfg.id: compare_revenue}
+)
 ```
 
 **Step 2:** Implement the comparison function (`compare_function()`) used above.
 
-_data_node_results_ is the list of the Output Data Nodes from all scenarios passed in the comparator. We iterate through it to compare scenarios.
+_revenues_ is the list of the Output (*revenues*) Data Nodes from all scenarios passed in 
+the comparator. We iterate through it to compare scenarios.
 
 ```python
-def compare_function(*data_node_results):
-    compare_result = {}
-    current_res_i = 0
-    for current_res in data_node_results:
-        compare_result[current_res_i] = {}
-        next_res_i = 0
-        for next_res in data_node_results:
-            print(f"comparing result {current_res_i} with result {next_res_i}")
-            compare_result[current_res_i][next_res_i] = next_res - current_res
-            next_res_i += 1
-        current_res_i += 1
-    return compare_result
+def compare_revenue(*revenues):
+    scenario_names = [f"Scenario {i}" for i in range(len(revenues))]
+    comparisons = {"Scenarios": scenario_names,
+                   "Revenues": list(revenues)}
+    return pd.DataFrame(comparisons)
```

Comment:
Much better!

Thx

Reply:
I think there is an API problem as I don't know which Scenario Ia m comparing in the compare function

Reply:
Indeed. That's a good point. 

The idea was to ask the user to define a comparison function as simple as possible. So we just provide the list of data to compare.

You are saying that you require more information than just the list of data. You are talking about the scenario, but the data node as well could be useful.

Can you create an issue about that?

Reply:
https://github.com/Avaiga/taipy/issues/1233

Reply:
Thank you.